Sources/GarageStorage/Garage+Codable.swift

```diff
@@ -314,6 +314,18 @@ extension Garage {
         return try makeSyncableObjects(from: coreDataObjects)
     }
     
+    /// Returns all the objects of type T conforming to Codable that have a given sync status
+    ///
+    /// - parameter syncStatus: The Sync Status
```

Comment:
I think(?) that you wanted the parameter name in this line to be `withStatus`, not `syncStatus`. Also, parameters are in the wrong order, if you care about that.

Reply:
Going to fix the parameter comment order. It looks like the parameter name is being documented from the callee point of view, rather than the caller, throughout the repo. Probably should be flipped, but going to keep syncStatus here for consistency. Dumb question: how do other repos document an anonymous parameter, like objectClass in this case?